src/main/scala/encry/storage/levelDb/versionalLevelDB/WalletVersionalLevelDB.scala

```diff
@@ -0,0 +1,110 @@
+package encry.storage.levelDb.versionalLevelDB
+
+import cats.instances.all._
+import cats.syntax.semigroup._
+import com.google.common.primitives.Longs
+import com.typesafe.scalalogging.StrictLogging
+import encry.modifiers.state.StateModifierSerializer
+import encry.modifiers.state.box.Box.Amount
+import encry.modifiers.state.box.EncryBaseBox
+import encry.modifiers.state.box.TokenIssuingBox.TokenId
+import encry.settings.{Constants, LevelDBSettings}
+import encry.storage.levelDb.versionalLevelDB.VersionalLevelDBCompanion._
+import encry.utils.{BalanceCalculator, ByteStr}
+import encry.utils.CoreTaggedTypes.ModifierId
+import io.iohk.iodb.ByteArrayWrapper
+import org.encryfoundation.common.Algos
+import org.encryfoundation.common.utils.TaggedTypes.ADKey
+import org.iq80.leveldb.{DB, ReadOptions}
+import scorex.crypto.hash.Digest32
+
+import scala.util.Success
+
+case class WalletVersionalLevelDB(db: DB, settings: LevelDBSettings) extends StrictLogging with AutoCloseable {
+
+  import WalletVersionalLevelDBCompanion._
+
+  val levelDb: VersionalLevelDB = VersionalLevelDB(db, settings)
+
+  def getAllBoxes: Seq[EncryBaseBox] = levelDb.getAll
+      .map { case (key, bytes) => StateModifierSerializer.parseBytes(bytes, key.head) }
```

Comment:
I think we have to filter out BALANCE_KEY, for example (and maybe something else)